src/modules/Dropdown/DropdownMenu.js

```diff
@@ -32,6 +35,7 @@ DropdownMenu.propTypes = {
 
   /** Classes to add to the className. */
   className: PropTypes.string,
-}
 
-export default DropdownMenu
+  /** Will mark the menu as opened when nested in another menu */
+  opened: PropTypes.bool,
```

Comment:
We do need this, thanks.  Let's call the prop `active` though so it is inline with SUI core:

http://semantic-ui.com/modules/dropdown.html#active